geant4_vmc.sh

```diff
@@ -1,9 +1,10 @@
 package: GEANT4_VMC
 version: "%(tag_basename)s"
-tag: "v5-3"
+tag: "v5-4"
 source: https://github.com/vmc-project/geant4_vmc
 requires:
   - ROOT
```

Comment:
We can remove ROOT as VMC already depends on ROOT

Reply:
removed